conans/client/tools/win.py

```diff
@@ -447,16 +446,11 @@ def vcvars_dict(settings, arch=None, compiler_version=None, force=False, filter_
     cmd = vcvars_command(settings, arch=arch,
                          compiler_version=compiler_version, force=force,
                          vcvars_ver=vcvars_ver, winsdk_version=winsdk_version, output=output)
-    cmd += " && echo __BEGINS__ && set"
```

Comment:
why removed `__BEGINS__`?

Reply:
Now we are piping the command to a file, so it would be:

```
... && echo __BEGINS__ && set > file.tmp

```

the `echo __BEGINS__` won't go to the `file.tmp` and it is not in output. We get rid of that trick.